### DISCUSSION TITLE
Wallet issue

### DISCUSSION THREAD

**Discussion by gazecloud** [Chia-Network/chia-blockchain#10182] [Support]
1. Wallet status cannot be synchronized,  always stay in:   Syncing (1,541,282)
2. Wallet receive address changed for changing pool , so 0.25xch can not get into my wallet.

**Comment by esaung** [comment]
Can you please provide more details?
which version of the app are you running? what operating system?

were you getting a pool payout?

**Comment by gazecloud** [comment]
OS is WIN10.
I change a PC, wallet status work well.
But for the lost 0.25 xch, may cause by changing pool between 
https://asia1.pool.space and https://pool.xchpool.org. 
I can blocks message:
"You have found 1 block

According to the Chia pooling protocol, 0.25 XCH is given directly to the farmer who finds a block. This reward does not go through Space Pool. If you can't find it, refer to this [post](https://blog.pool.space/how-to-find-your-0-25xch-after-winning-a-block-in-the-pool-fc61d0bee5c5).

Date
2022/1/18 22:27
Block #
1439771
Hash
5d1d9421d71fe8e907170c37504831d9a6575a9aebcb8ff8c66ac78f80429076"

so how can I get it?

**Comment by loppefaaret** [comment]
if you won a block, using a pool, with a plotNFT - the 0.25 would be directly rewarded to your "farmer reward target address" which you can find in you Chia client.
on the "Farm" tab, in the top right, under the 3 `...` menu - there is a farmer and a pool payout address. the 0.25 would go to whatever address is set as the "farmer rewards taget address"
if you want, you can take any receive address from any wallet you wish, and set it as the "farmer reward target address"

**Comment by gazecloud** [comment]
Current situation is I can see Jan.18  I have get the rewards, but because I have change my wallet address to a new one, so the rewards had been send to the old address, not to my current address. 


How can I get it back?

**Comment by gazecloud** [comment]
I can understand later if I get a reward, the 0.25 will go to the new address. It's right. but how to do with the old rewards ?

**Reply by loppefaaret** [reply]
if the address was belonging to your wallet, it would always and forever lead to your wallet - if it did not show up in your wallet, it must have been an address that did not belong to your wallet - and in turn that means that you cannot control those 0.25 from the 18th of JAN.

**Reply by gazecloud** [reply]
In fact, I own only one private wallet.  but in chia program "Wallets" menu, I try to new receive address then set the the address to which you have told me  "on the "Farm" tab, in the top right, under the 3 ... menu - there is a farmer and a pool payout address. the 0.25 would go to whatever address is set as the "farmer rewards taget address""

I guess there maybe some bugs when changing pool or new recive address  will cause the rewards cannot be received?

**Reply by gazecloud** [reply]
"Note that this does not change your pooling payout addresses. This only affects old format plots, and the 0.25XCH reward for pooling plots."

**Reply by gazecloud** [reply]
<html>
<body>


01/18/2022, 10:27:43 PM | 🌱 Farmer reward coin |   | 0.25 XCH | $19.9 | 1,439,778 | 0Unspent
-- | -- | -- | -- | -- | -- | --



</body>
</html>[](https://www.chiaexplorer.com/blockchain/coin/undefined)01/18/2022, 10:27:43 PM	🌱 Farmer reward coin		0.25 XCH	$19.9	1,439,778	0Unspent

**Reply by gazecloud** [reply]
Foliage Sub Block
foliage_block_hash
0x404e8f56204cfc93498ab44150809aa95439c5740a1d19cf7c56e51e2aa4be2a
prev_block_hash
0xae8d7c85592a6df13f8427653ec32213b64ae73ebf7c0165987403015c9d02a3
reward_block_hash
0x36aca30cef58c9e309f56b2eba1304af29b474d6d69eb18b8e08bfdbfd70536c
extension_data
0x0000000000000000000000000000000000000000000000000000000003a2c7c9
farmer_reward_puzzle_hash
0x171abb80a587333076af35483443bd99d76ddf4b84c1a461e120f918c6e6405f
farmer_reward_address
[xch1zudthq99suenqa40x4yrgsaan8tkmh6tsnq6gc0pyru333hxgp0s5z5340](https://www.chiaexplorer.com/blockchain/address/xch1zudthq99suenqa40x4yrgsaan8tkmh6tsnq6gc0pyru333hxgp0s5z5340)
unfinished_reward_block_hash
0x404e8f56204cfc93498ab44150809aa95439c5740a1d19cf7c56e51e2aa4be2a
pool_target.max_height
0
pool_target.puzzle_hash
0xd52f1f29d565878b9a1c02193d6dea943b117b37ca59601dc029e21457e3f57a
pool_target_address
[xch165h372w4vkrchxsuqgvn6m02jsa3z7ehefvkq8wq983pg4lr74aqn07x0c](https://www.chiaexplorer.com/blockchain/address/xch165h372w4vkrchxsuqgvn6m02jsa3z7ehefvkq8wq983pg4lr74aqn07x0c)

**Comment by jirolu16** [comment]
![image](https://user-images.githubusercontent.com/58925764/154298668-5a14c292-6482-4587-96d9-5f49afd3032e.png)
how to fix this?

**Reply by gazecloud** [reply]
wait or use a better computer

**Reply by jirolu16** [reply]
whats wrong about my PC? 
Device name	jirolu-PC
Processor	Intel(R) Core(TM) i5-10400 CPU @ 2.90GHz   2.90 GHz
Installed RAM	16,0 GB (15,9 GB usable)
Device ID	
Product ID	
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display
Edition	Windows 11 Pro
Version	21H2
Installed on	‎12/‎02/‎2022
OS build	22000.493
Experience	Windows Feature Experience Pack 1000.22000.493.0
